goldstone/core/models.py

```diff
@@ -2634,3 +2636,70 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class DefinedSearch(models.Model):
+    """Defined searches, both system and user-created."""
+
+    # This object's Goldstone UUID.
+    uuid = UUIDField(version=1, auto=True, primary_key=True)
+
+    name = models.CharField(max_length=64)
+
+    desc = models.CharField(max_length=1024)
```

Comment:
The Django idiom is to use full words for field names, unless it's prohibitive.  So, this should be "description".